src/NosCore.Packets/ClientPackets/Player/BpmPacket.cs

```diff
@@ -0,0 +1,34 @@
+﻿//  __  _  __    __   ___ __  ___ ___
+// |  \| |/__\ /' _/ / _//__\| _ \ __|
+// | | ' | \/ |`._`.| \_| \/ | v / _|
+// |_|\__|\__/ |___/ \__/\__/|_|_\___|
+// -----------------------------------
+
+using NosCore.Packets.Attributes;
+using NosCore.Packets.Enumerations;
+using System.Collections.Generic;
+
+namespace NosCore.Packets.ClientPackets.Player
+{
+    [PacketHeader("bpm", Scope.InGame)]
+    public class BpmPacket : PacketBase
+    {
+        [PacketIndex(0)]
+        public byte Unknow { get; set; } = 70; // todo : to find
+
+        [PacketIndex(1)]
+        public bool IsBattlePassIconEnabled { get; set; }
+
+        [PacketIndex(2)]
+        public int MaxBattlePassPoints { get; set; }
+
+        [PacketIndex(3)]
+        public int Unknow2 { get; set; } = 21093011; // todo : to find
+
+        [PacketIndex(4)]
+        public int Unknow3 { get; set; } = 21111111; // todo : to find
+
+        [PacketIndex(5)]
+        public List<BpmSubTypePacket>? QuestList { get; set; }
```

Comment:
can this be null ?

Reply:
Yeah, on official if it's null there, no missions are showed up